grakn-engine/src/main/java/ai/grakn/engine/data/RedisWrapper.java

```diff
@@ -150,7 +150,7 @@ public RedisWrapper build() {
             if (useSentinel) {
                 jedisPool = new JedisSentinelPool(masterName, uriSet, poolConfig, TIMEOUT);
             } else {
-                String uri = uriSet.stream().findFirst().get();
+                String uri = uriSet.iterator().next();
```

Comment:
If we want a single element then`Iterables.getOnlyElement(uriSet)` might be useful.

Reply:
That will throw an exception is there is more than one element in the collection, so I can't use it here.